content/quiz_events.md

```diff
@@ -4,21 +4,134 @@
 
 ## Rulings
 
+### Context
+: Shall be limited to five verses before or after the verse. On all question types, a quizzer cannot automatically be ruled out of context by a single word, even if it is a unique word.  Rather, the quizzer must say a complete thought or phrase that conclusively puts them out of context.
+
+### Interchangeable
+: Means not only not incorrect; means either are acceptable as correct.
+
+### Special Name
+: A name which imparts specific character qualities or attributes to that person of the deity.
+
+### Deity Rule
+- The correct person of the deity must be given. 
+- The quizzer may not move between persons of the deity when answering. Giving the name of a deity that is not correct is considered giving incorrect information.
+- If a quizzer refers to any person of the deity as either “God” or “Lord”, these must be clarified by the quizzer within 30 seconds when the text requires a more specific answer. 
+- “Jesus” and “Christ” are Interchangeable (but refer to the person of the Son) and the quizzer will be called correct.
+- If a pronoun is used in the text to name the deity, any of the names for that person of the deity within context may be considered an acceptable answer when clarifying.
+    - If the pronoun naming the deity is identified by the quizzer using a name that correctly identifies the deity, but with an identification that is out of context, the quizzer will be considered to have not stayed in context.
+- When there is a Special Name of the deity given in the text, the Special Name must be given in the quizzer’s answer.  (i.e.  “The Lord God Almighty”, “The Bread of Life”, “Spirit of the Living God”.)
+    - If the quizzer states any other Special Names which are not in context, the quizzer will be considered to have not stayed in context, even if they refer to the same person of the deity. Example: Jesus refers to himself as “the bread of life” in John 6 and “the good shepherd” in John 10. 
+- When a quizzer gives other names for the same person of the deity, used within context, these should be allowed to be clarified within 30 seconds when a more specific answer is needed.
+
 ### Correct
+The quizzer's response is correct when:
+- It fulfills all requirements specific to the Question Type.
```

Comment:
I added this line, which enables me to remove repeats of aspects of a correct answer (e.g. word-perfect on F/Q)